packages/viewer/src/components/scene-tree/scene-tree.css

```diff
@@ -10,6 +10,8 @@
   width: 300px;
   height: 100%;
   user-select: none;
+  -webkit-user-select: none;
```

Comment:
We use `user-select` in a few places. I think we should investigate adding [postcss/autoprefixer](https://www.npmjs.com/package/@stencil/postcss) to our stencil builds, so we don't have to manually manage prefixes.